packages/cli/test/config.test.js

```diff
@@ -0,0 +1,52 @@
+import assert from 'node:assert'
+import { describe, test } from 'node:test'
```

Comment:
I remember `node:test` has some problems on croess-platform. You could use `vitest` under the `packages/rolldown`.